CHANGELOG.md

```diff
@@ -5,6 +5,42 @@ All notable changes for each version of this project will be documented in this
 ## 12.1.0
 
 ### New Features
+- Added `IgxAccordion` component
+    - A collection of vertically collapsible igx-expansion panels that provide users with data and the ability to drill into it in a compact manner. The control is **not** data bound and takes a declarative approach, giving users more control over what is being rendered.
```

Comment:
I'm not sure what **drill into it in a compact manner** has to mean. Can we think of another way to say it?